packages/base-styles/_variables.scss

```diff
@@ -93,8 +93,12 @@ $border-width-tab: 1.5px;
 $helptext-font-size: 12px;
 $radius-round: 50%;
 $radius-block-ui: 2px;
-$radio-input-size: 20px;
+$radio-input-size: 16px;
 $radio-input-size-sm: 24px; // Width & height for small viewports.
+$toggle-width: $grid-unit-40;
+$toggle-height: $grid-unit-20;
+$toggle-border-width: 1px;
+$toggle-thumb-size:  $grid-unit-15;
```

Comment:
Is it necessary to lift all these variables up into `base-styles`? I'm not sure how conservative we are about adding global variables, but personally I wouldn't add them if it isn't necessary to share them across packages.

If it's just to share values between the FormToggle and ToggleControl components, we can keep them within the `@wordpress/components` package.

Reply:
Agreed. On a separate note, at some point we should look at tidying up shared styles in the package: some variables are defined in Sass, some as CSS custom properties, and some in JS.

Reply:
Moved back to the FormToggle styles in 0d73cc5. (Ideally this should use a `@use` to scope the variables better, but we're not set up to do that properly right now.)